packages/client/src/rpc/modules/engine/engine.ts

```diff
@@ -292,26 +289,6 @@ export class Engine {
       () => this.connectionManager.updateStatus(),
     )
 
-    /**
-     * exchangeTransitionConfiguration
-     */
-    this.exchangeTransitionConfigurationV1 = cmMiddleware(
```

Comment:
Had to check, but we can indeed remove this method https://github.com/ethereum/execution-apis/blob/b7432c54d2faa5f282db7b3202c3c2ef907103ad/src/engine/cancun.md#deprecate-engine_exchangetransitionconfigurationv1

Reply:
Yes, also looked this up before removal (not sure why I stumbled initially upon that, likely because it used the functionality we are removing here).